app/config/sources.inc.php

```diff
@@ -437,8 +439,18 @@
             'it', 'kab', 'ko', 'pl', 'pt-BR', 'sq', 'sv-SE', 'uk', 'zh-TW',
         ],
     ],
+    'mozorg/internet-health/decentralization.lang' => [
+        'deadline' => '2017-05-24',
+        'flags'    => [
+            'opt-in' => ['all'],
+        ],
+        'priority'          => 3,
```

Comment:
If you want this file to be 3, you can remove this line, it will get 3 by default